docs/en_us/platform_api/source/course_catalog/catalogs.rst

```diff
@@ -0,0 +1,315 @@
+.. _Course Catalog API Catalogs Resource:
+
+########################################
+Course Catalog API Catalogs Resource
+########################################
+
+With the Course Catalog API **Catalogs** resource, you can complete the
+following tasks.
+
+   * :ref:`Get a list of all available course catalogs <Get a List of All
+     Course Catalogs>`
+     - GET
+     - /api/v1/catalogs/
+   * - :ref:`Get information about a specific catalog <Get Information About a
+     Specific Catalog>`
+     - GET
+     - /api/v1/catalogs/1/
+   * - :ref:`Get a list of all courses in a catalog <Get a List of All Courses
+     in a Catalog>`
+     - GET
+     - /api/v1/catalogs/1/courses/
+
+.. contents::
+   :local:
+   :depth: 1
+
+.. _Get a List of All Course Catalogs:
+
+**************************************
+Get a List of All Course Catalogs
+**************************************
+
+The endpoint to get a list of all course catalogs is ``/api/v1/catalogs/``.
+
+=====================
+Use Case
+=====================
+
+Get a list of all the available course catalogs.
+
+=====================
+Example Request
+=====================
+
+``GET /api/v1/catalogs/``
+
+=====================
+Response Values
+=====================
+
```

Comment:
These values came from https://stage-edx-discovery.edx.org/api/v1/catalogs/. Please confirm that they're correct.